0000-template.md

```diff
@@ -1,84 +0,0 @@
-- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
```

Comment:
you should keep the template around

Reply:
fixed, forgot completely about that while renaming the file